src/app/backend-api.service.ts

```diff
@@ -375,22 +276,12 @@ export class NFTEntryResponse {
   BuyNowPriceNanos: number;
 }
 
-export class NFTBidEntryResponse {
-  PublicKeyBase58Check: string;
-  ProfileEntryResponse: ProfileEntryResponse;
-  PostHashHex: string;
-  PostEntryResponse: PostEntryResponse | undefined;
-  SerialNumber: number;
-  BidAmountNanos: number;
-
-  HighestBidAmountNanos: number | undefined;
-  LowestBidAmountNanos: number | undefined;
-
-  BidderBalanceNanos: number;
-
+// TODO: fix this type. The additional fields here are added client side
+// and are not part of the backend schema.
+export type NFTBidEntryResponse = NFTBidEntry & {
```

Comment:
should we update deso-protocol-types?

Reply:
Imo it should not be a part of the types library because it includes custom fields added by the diamond frontend. It wouldn't make sense to add as part of the backend type definition. It should be some composite type defined in the diamond frontend. I wouldn't worry with it personally.